src/query.rs

```diff
@@ -475,6 +481,8 @@ macro_rules! add_lints {
     }
 }
 
+// The following add_lints! invocation is programmatically edited by scripts/make_new_lint.sh
+// If you must manually edit it, be sure to read the "Requirements" comments in that script first
```

Comment:
This comment is great and I'd love to merge it right away. If you'd like, I can merge it in its own PR if you were to open one for it.